### PR TITLE
Adds pick-up, drop and throw sounds for items.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd & ActionNinja
 soundadd: Some things now make sounds when picked up, thrown, or dropped.
/:cl:

https://streamable.com/koz4r

### CODE REVIEW

**Comment by PKPenguin321** [comment]
neat

**Comment by optimumtact** [comment]
source?

**Comment by Qustinnus** [comment]
knowing action ninja, homemade.

**Comment by Krysonism** [comment]
Based

**Comment by dr-bee** [comment]
All hail Floyd, king of immersion!

**Comment by blessedmulligan** [comment]
are pickup sounds audible to nearby players?

**Comment by Qustinnus** [comment]
Ya

**Comment by cacogen** [comment]
Judging from the code they don't go through walls though so maint bases should be safe

**Comment by blessedmulligan** [comment]
still a nerf to stealing things from distracted people. Definitely worth it though

**Comment by actioninja** [comment]
I made all these sounds a while ago and then the implementation was annoying me and I intended to make more but then never did.
At least they'll finally see a use.

File: code/__DEFINES/sound.dm [hunk 1]
```diff
@@ -77,3 +77,6 @@
 
 #define INTERACTION_SOUND_RANGE_MODIFIER -3
 #define EQUIP_SOUND_VOLUME 30
+#define PICKUP_SOUND_VOLUME 40
+#define DROP_SOUND_VOLUME 50
+#define YEET_SOUND_VOLUME 90
```
Comment (fludd12) [inline code/__DEFINES/sound.dm]: `n o`

File: code/game/objects/items.dm [hunk 2]
```diff
@@ -37,9 +37,14 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 	var/hitsound
 	var/usesound
-	var/throwhitsound
+	///Used when yate into a mob
```
Comment (fludd12) [inline code/game/objects/items.dm]: Yate? ___YATE?___ If you're going to do this, at least use the proper past tense `yote`.

**Review by fludd12 — COMMENTED** [review]
👌

File: code/game/objects/items.dm (hunk 2 above)
Comment (Qustinnus) [reply code/game/objects/items.dm]: eat - ate
yeet - yate

Comment (Qustinnus) [reply code/game/objects/items.dm]: phonetically the same, I've debatted this with people many times so come at me

**Comment by Iatots** [comment]
That's a really UNGA way of picking up a piece of paper jesus christ.

File: code/game/objects/items.dm (hunk 2 above)
Comment (cacogen) [reply code/game/objects/items.dm]: greet - greeted
yeet - yeeted

:^)

Comment (Qustinnus) [reply code/game/objects/items.dm]: I feel like eat is closer so go away

Comment (fludd12) [reply code/game/objects/items.dm]: it's not yeat, though

Comment (Qustinnus) [reply code/game/objects/items.dm]: bro its phonetic ok?

File: code/__DEFINES/sound.dm (hunk 1 above)
Comment (ATH1909) [reply code/__DEFINES/sound.dm]: `y e s`

**Comment by Ty-the-Smonk** [comment]
Can’t wait to go into hop line and have clown spam dropping horns

File: code/game/objects/items.dm [hunk 3]
```diff
@@ -1,835 +1,860 @@
-GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/effects/fire.dmi', "fire"))
```
Comment (optimumtact) [inline code/game/objects/items.dm]: was it necessary to change the line endings of the entire file?

Comment (Qustinnus) [reply code/game/objects/items.dm]: fucking web edits hold on

**Review by actioninja — CHANGES_REQUESTED** [review]

**Comment by Qustinnus** [comment]
@actioninja this rdy

**Review by actioninja — APPROVED** [review]
I don't like merging shit I have personal involvement in but this is ready

**Pull request merged**